cmd/libs/go2idl/generator/snippet_writer.go

```diff
@@ -116,6 +116,10 @@ func (s *SnippetWriter) Do(format string, args interface{}) *SnippetWriter {
 	return s
 }
 
+func (s *SnippetWriter) Out() io.Writer {
```

Comment:
Using it to get a writer I can use with Fprintf. The proto idl generator
doesn't benefit much from snippet writer in general, and rather than make
up a new format mechanism I chose to use straight Fprintf.  In general, for
something moderately complex like the protogen path the code for Fprintf is
more readable than snippet writer.

On Nov 27, 2015, at 5:22 AM, Wojciech Tyczynski <notifications@github.com>
wrote:

In cmd/libs/go2idl/generator/snippet_writer.go
<https://github.com/kubernetes/kubernetes/pull/17853#discussion_r46033964>:

> @@ -116,6 +116,10 @@ func (s *SnippetWriter) Do(format string, args interface{}) *SnippetWriter {
>  	return s
>  }
>
> +func (s *SnippetWriter) Out() io.Writer {

why do you need it?

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/17853/files#r46033964>.